stable/sonarqube/templates/deployment.yaml

```diff
@@ -22,6 +22,15 @@ spec:
 {{ toYaml .Values.securityContext | indent 8 }}
       {{- if .Values.plugins.install }}
       initContainers:
+        - name: init-sysctl
+          image: busybox:1.31
+          command:
+          - sysctl
+          - -w
+          - vm.max_map_count=262144
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
```

Comment:
Is this strictly required? Our organization enforces strict PodSecurityPolicies which prevent us from being able to upgrade to this version

```
Events:
  Type     Reason        Age                   From                   Message
  ----     ------        ----                  ----                   -------
  Warning  FailedCreate  90s (x17 over 6m58s)  replicaset-controller  Error creating: pods "upgrade-sonarqube-584cb87dd5-" is forbidden: unable to validate against any pod security policy: [spec.initContainers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed spec.initContainers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed]
```

Reply:
@jcoeltjen

Reply:
@robbie-demuth The privileged flag is required to use sysctl to set **vm.max_map_count**. 
As long as sonarqube uses an elasticsearch version that requires this setting you have to set this.
But maybe we can make the whole init container optional to be able to use the Chart on secure clusters. But then you either have to use node selectors to select nodes with matching sysctl configurations or you have to reconfigure all your nodes in your cluster.
Still having the option to do so would solve the problem for advanced users.

Reply:
It looks like our cluster nodes have sufficient mmap count limits, so we good. I just saw your PR for disabling the init container. Thanks for tackling this so quickly!

Reply:
Of course. That was just a small change and I was honestly surprised that the privileged init container did not cause problems for more people.
Hopefully this fixes the issue for you.